data/boarding missions.txt

```diff
@@ -61,6 +61,160 @@ mission "Assisting Merchant (Large)"
+				launch
+			label fire
+			`	The shots you fire quickly break the asteroid into pieces, which keep following the same course. Before you can even try to destroy them, most have already hit the <origin>, which is now quickly depressurizing and losing the hull's integrity. There's not much you can do anymore, except to try to move a safe distance from the exploding ship...`
+				launch
+			label repair
+			`	You dock with the <origin>, hoping to get out of this alive, but as soon as you reach the hatch, the asteroid hits and a series of explosions start coming from the disabled ship. You are able to reach your cockpit and undock, but the crew on the other ship couldn't reach yours in time...`
+				launch
+			label move
+			`	The asteroid, after hitting your shield, breaks into a million fragments which fly away from the <origin>. After repairing it, you hail its captain, which thanks you with a payment of 30,000 credits.`
+				decline
+	on decline
+		payment 30000
+
+mission "Asteroid Course B"
+	assisting
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	destination "Earth"
+	on offer
+		conversation
+			`As you are approaching the <origin>, your radar sensors detect that the ship you are assisting is in the middle of a large asteroid's course! While in a normal flight this wouldn't cause much trouble, a ship without active shields and that has already suffered heavy damages could be completely destroyed.`
+			branch unarmed
+				"armament deterrence" == 0
+			choice
+				`	(Shoot towards the asteroid to destroy it.)`
+					goto fire
+				`	(Move your ship between the asteroid and the <origin>, then use your shields to protect it.)`
+					goto move
+				`	(Try to quickly repair the <origin>, so that it can move away from the asteroid's course.)`
+					goto repair
+				`	(Interrupt the docking maneuvers and fly away.)`
+					goto escape
+			label unarmed
+			choice
+				`	(Move your ship between the asteroid and the <origin>, then use your shields to protect it.)`
+					goto move
+				`	(Try to quickly repair the <origin>, so that it can move away from the asteroid's course.)`
+					goto repair
+				`	(Interrupt the docking maneuvers and fly away.)`
+			label escape
+			`	You turn your ship around and activate your thrusters, trying to get as far as you can from the disabled ship. A few seconds later, the asteroid tears the <origin> apart, killing all of its crew members.`
+				launch
+			label fire
+			`	The shots you fire quickly break the asteroid into pieces, which seem to have changed course enough to completely miss the <origin>. After repairing it, you hail its captain, which thanks you with a payment of 30,000 credits.`
+				decline
```

Comment:
Instead of having an entirely separate mission that only changes this single outcome, you can put the following under `label fire` in the first mission.
```
branch success
	random > 50
```